### PR TITLE
Delete the route to display all statuses

### PR DESCRIPTION
**Pull request by ferdynton**
It does not have any sort of pagination so it tries to fetch all
statuses from Redis, which is potentially a very expensive operation
depending on the number of elements.

In avant-basic this feature is completely broken because the number of
stored statuses is very large so there's no way Redis has time to get
them all in the 30 seconds that our app has to respond by default.
Because of this, there's no harm in removing this for us right now and
there's a chance it helps alleviating our apparently random Redis
issues.

### CODE REVIEW

File: lib/sidekiq-status/web.rb [hunk 1]
```diff
@@ -37,59 +37,6 @@ def status_label(status)
...
           end
         end
-
-        def has_sort_by?(value)
-          ["worker", "status", "update_time", "pct_complete", "message"].include?(value)
-        end
-      end
-
-      app.get '/statuses' do
-        namespace_jids = Sidekiq.redis{ |conn| conn.keys('sidekiq:status:*') }
```
Comment (ferdynton) [inline lib/sidekiq-status/web.rb:47]: Maybe this was it? @gabelerner

**Comment by elech** [comment]
LGTM.

On a serious note. I don't really care but I assume you guys don't want me having access here anymore.

**Comment by ferdynton** [comment]
@elech hey Eric! Long time no see! Well, it's a public fork, so I don't think we care much about who sees this or not `¯\_(ツ)_/¯`

**Comment by gabelerner** [comment]
Yea this would do it if hit ... Lgtm

**Comment by elech** [comment]
Hey man. My mistake, in that case carry on!

**Pull request merged**